src/task/filterPackages.ts

```diff
@@ -48,6 +59,13 @@ export function filterPackages(options: {
     filtered = filtered.concat(getTransitiveConsumers(filtered, allPackages));
   }
 
+  // adds dependencies of all filtered package thus far
+  if (includeDependencies) {
+    filtered = filtered.concat(
+      getTransitiveDependencies(filtered, allPackages)
```

Comment:
is there some ignore option in the config we need to respect here as well?

Reply:
lage doesn't currently support an `ignore` option for ignoring packages.
lage's [ `--ignore`](https://microsoft.github.io/lage/guide/cli.html#ignore) option is for calculating changes in combination with the `--since` flag.